ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -167,6 +167,24 @@ const limitCutIds: readonly string[] = Object.keys(limitCutMap);
 const wingIds: readonly string[] = Object.values(wings);
 const superchainNpcBaseIds: readonly string[] = Object.values(superchainNpcBaseIdMap);
 
+const whiteFlameDelayOutputStrings = {
+  delay1: {
+    en: 'now',
```

Comment:
In some languages such expressions are ambiguous. There are no sufficient and clear words to describe these 5 delayed states. Generally speaking, problems often occur in group [1, 3] and [2, 4] group, which can be written to `wait for [1, 3] bait` when [2, 4] need for the bait after [1, 3] bite.

Reply:
It's ambiguous in English as well, and I don't know that these really differentiate it.  It's more just a very rough indication of how much waiting is required without just saying the numbers 1-5, which I think would be less understandable.

Reply:
I think these are every ~3s apart, so one alternative is just to say 3 seconds, 6 seconds, 9 seconds, etc.